src/pages/workspace/accounting/intacct/export/SageIntacctReimbursableExpensesPage.tsx

```diff
@@ -0,0 +1,136 @@
+    const {reimbursable, reimbursableExpenseReportDefaultVendor} = policy?.connections?.intacct?.config?.export ?? {};
+
+    const data: MenuListItem[] = Object.values(CONST.SAGE_INTACCT_REIMBURSABLE_EXPENSE_TYPE).map((expenseType) => ({
+        value: expenseType,
+        text: translate(`workspace.sageIntacct.reimbursableExpenses.values.${expenseType}`),
+        keyForList: expenseType,
+        isSelected: reimbursable === expenseType,
+    }));
+
+    const headerContent = useMemo(
+        () => (
+            <View>
+                <Text style={[styles.ph5, styles.pb5]}>{translate('workspace.sageIntacct.reimbursableExpenses.description')}</Text>
+            </View>
+        ),
+        [translate, styles.pb5, styles.ph5],
+    );
+
+    const selectReimbursableDestination = useCallback(
+        (row: MenuListItem) => {
+            if (row.value !== reimbursable) {
+                updateSageIntacctReimbursableExpensesExportDestination(policyID, row.value);
+            }
+            if (row.value === CONST.SAGE_INTACCT_REIMBURSABLE_EXPENSE_TYPE.VENDOR_BILL) {
+                Navigation.goBack(ROUTES.POLICY_ACCOUNTING_SAGE_INTACCT_EXPORT.getRoute(policyID));
+            }
+        },
+        [reimbursable, policyID],
+    );
+
+    const defaultVendorSection = {
+        description: translate('workspace.sageIntacct.defaultVendor'),
+        action: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_SAGE_INTACCT_DEFAULT_VENDOR.getRoute(policyID, 'reimbursable')),
+        title: reimbursableExpenseReportDefaultVendor
+            ? getDefaultVendorName(reimbursableExpenseReportDefaultVendor, intacctData?.vendors ?? [])
+            : translate('workspace.sageIntacct.notConfigured'),
+        hasError: !!config?.export?.errorFields?.reimbursableExpenseReportDefaultVendor,
+        pendingAction: config?.export?.pendingFields?.reimbursableExpenseReportDefaultVendor,
+    };
+
+    const defaultVendor = (
+        <OfflineWithFeedback
+            key={defaultVendorSection.description}
+            pendingAction={defaultVendorSection.pendingAction}
+        >
+            <MenuItemWithTopDescription
+                title={defaultVendorSection.title}
+                description={defaultVendorSection.description}
+                shouldShowRightIcon
+                onPress={defaultVendorSection.action}
+                brickRoadIndicator={defaultVendorSection.hasError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
+            />
+        </OfflineWithFeedback>
+    );
+
+    return (
+        <ScreenWrapper
+            includeSafeAreaPaddingBottom={false}
+            testID={SageIntacctReimbursableExpensesPage.displayName}
+        >
+            <HeaderWithBackButton
+                title={translate('workspace.sageIntacct.reimbursableExpenses.label')}
+                onBackButtonPress={() => Navigation.goBack(ROUTES.POLICY_ACCOUNTING_SAGE_INTACCT_EXPORT.getRoute(policyID))}
```

Comment:
Same here, can we use ConnectionLayout or SelectionScreen instead?

Reply:
SelectionScreen can't be used here because we have some custom logic (toggle) but I can use ConnectionLayout for sure